services/tenant-ui/frontend/src/components/oca/Oca.vue

```diff
@@ -20,7 +20,7 @@
       <template #header>
         <div class="flex justify-content-between">
           <div class="flex justify-content-start">
-            <CreateOca />
+            <CreateOca v-if="config.frontend.showWritableComponents" />
```

Comment:
I would consider leaving OCA management for production "read-only" cases. It's not related to Line of Business data or credential management. It's almost more of a Tenant Setting thing.
This allows users to set a specified OCA bundle with a credential definition they manage. So a "traction ecosystem" can be built up where known holders can determine which OCA from a known issuer they are meant to display. Something that seems like settings management through the Tenant UI to me I guess.

Could be more discussion, simple to change in or out one way or another with this new flag.

Reply:
I am thinking we hide this until we have a better "common" OCA set-up: this wouldn't affect outbound credentials, but only credentials stored in the wallet and while the UI would make them prettier I am not too sure users would go in and associate a bundle to their credentials manually. Plus it can be easily reactivated if/when deemed necessary, as you mentioned.

Reply:
OK... the button is hidden now for everyone.